### ISSUE TITLE
sketchybar: could not initialize daemon! abort..

### ISSUE THREAD

**Issue by tim0120** [FelixKratz/SketchyBar#238]
When I run sketchybar, I get this error:
```sketchybar: could not acquire lock-file! abort..```

**Comment by tim0120** [comment]
I also can't find a `~/.config/sketchybar` folder, that should exist right?

**Comment by tim0120** [comment]
I've done the steps in [this issue](https://github.com/FelixKratz/SketchyBar/issues/234) but to no avail

**Comment by FelixKratz** [comment]
When sketchybar starts, it creates/ 
acquires a lock-file. Once you try to start a second instance it can not acquire the lock-file and will fail to launch a second time. If you remove the lock-file by force, as stated in the linked comment, you will allow the second process to progress beyond this check. But if sketchybar is already running the bootstrap name will already be in use and the second sketchybar task will fail to initialize zhe daemon.

Long story short:
sketchybar is already running, don’t remove the lock-file. You need to copy the demo config yourself after the brew install (commands are displayed in the caveats section) as per the setup instructions.

**Comment by tim0120** [comment]
@FelixKratz sorry, totally missed the caveats section, apologies. Thanks for the help, cheers!
